### PR TITLE
Semantic model configs

### LINKED ISSUE #4180: Supported configs (`meta`, `enabled`, `group`) for semantic models
At a minimum, update body to include a link to the page on docs.getdbt.com requiring updates and what part(s) of the page you would like to see updated.

Relevant PRs: https://github.com/dbt-labs/dbt-core/pull/8754, https://github.com/dbt-labs/dbt-core/pull/8766

Pages to update:
- https://docs.getdbt.com/reference/resource-configs/meta
- https://docs.getdbt.com/reference/resource-configs/enabled
- https://docs.getdbt.com/reference/resource-configs/group
- https://docs.getdbt.com/reference/resource-properties/config
- https://docs.getdbt.com/docs/build/semantic-models

New things for `semantic_models`:
- `meta` can now be nested under a `config` key (config added in [#8754](https://github.com/dbt-labs/dbt-core/pull/8754))
- `enabled` can now be nested under a `config` key (config added in [#8502](https://github.com/dbt-labs/dbt-core/pull/8502))
- `group` can now be nested under a `config` key (config added in [#8502](https://github.com/dbt-labs/dbt-core/pull/8502))

### Examples

[This](https://docs.getdbt.com/reference/resource-configs/meta) page should add a "Semantic models" tab that includes an example like the following:

`models/semantic_models.yml`
```yaml
semantic_models:
  - name: semantic_people
    model: ref('people')
    config:
      meta: {<dictionary>}
```

And [this](https://docs.getdbt.com/reference/resource-configs/enabled) page should have an example like this:

`models/semantic_models.yml`
```yaml
semantic_models:
  - name: semantic_people
    model: ref('people')
    config:
      enabled: false
```

And [this](https://docs.getdbt.com/reference/resource-configs/group) page include examples like this:

`schema.yml`
```yaml
semantic_models:
  - name: model_name
    config:
      group: finance
```

`dbt_project.yml`
```yaml
semantic-models:
  [<resource-path>](resource-path):
    +group: finance
```

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Addresses issue https://github.com/dbt-labs/docs.getdbt.com/issues/4180

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/reference/resource-configs/enabled.md [hunk 1]
```diff
@@ -250,6 +251,29 @@ exposures:
...
 
+<TabItem value="semantic models">
+
+<File name='semantic_models.yml'>
+
+```yml
+semantic_models:
+  - name: semantic_people
+    model: ref('people')
+    config:
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/enabled.md:270]: @matthewshaver should this be versioned as well? It feels odd to show the config option for semantic models if it's only available in 1.7+

File: website/docs/reference/resource-configs/enabled.md [hunk 2]
```diff
@@ -250,6 +251,29 @@ exposures:
...
+
+The `enabled` configuration can be nested under the `config` key.
+
+</VersionBlock>
+
+</TabItem>
+
 </Tabs>
 
 ## Definition
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/enabled.md:285]: Should the following definition now include metrics, exposures and semantic models?

File: website/docs/reference/resource-configs/meta.md [hunk 3]
```diff
@@ -172,6 +173,28 @@ exposures:
 
 </TabItem>
 
+<TabItem value="semantic models">
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/meta.md:177]: Same versioning question here

**Review by runleonarun — COMMENTED** [review]
Hey @matthewshaver IS there any way to version the semantic models tabs here? The floating versioned text is a little confusing.

File: website/docs/reference/resource-configs/enabled.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/reference/resource-configs/enabled.md:270]: @runleonarun  The semantic model was backported to previous versions, it's only the ability to nest the configurations that were added in 1.7 (from what I read in the original PRs)

Comment (runleonarun) [reply website/docs/reference/resource-configs/enabled.md:270]: @matthewshaver  so should we version the nested config key in the example?

**Review by runleonarun — COMMENTED** [review]
A question about the nested config key in examples

File: website/docs/reference/resource-configs/enabled.md (hunk 1 above)
Comment (matthewshaver) [reply website/docs/reference/resource-configs/enabled.md:270]: Good question. Let me see how the SME feels about it

File: website/docs/reference/resource-configs/enabled.md (hunk 2 above)
Comment (matthewshaver) [reply website/docs/reference/resource-configs/enabled.md:285]: Added it!

File: website/docs/reference/resource-configs/meta.md (hunk 3 above)
Comment (matthewshaver) [reply website/docs/reference/resource-configs/meta.md:177]: Changed versioning to whole section

File: website/docs/reference/resource-configs/enabled.md [hunk 4]
```diff
@@ -250,10 +251,39 @@ exposures:
 
 </TabItem>
 
+<TabItem value="semantic models">
+
+<VersionBlock lastVersion="1.7">
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/enabled.md]:
```suggestion
<VersionBlock lastVersion="1.6">
```

**Review by runleonarun — APPROVED** [review]
One tiny typo fix then 🚢 it!

**Pull request merged**